src/test/java/dynamo/RidesDaoTest.java

```diff
@@ -56,5 +64,53 @@ public void deleteTable() {
 			System.err.println(e.getMessage());
 		}
 	}
+	
+	public void insertTest(String email, String name, int year, String phoneNumber, String church, boolean attendance) {

```

Comment:
Oops I should probably teach you how to write unit tests haha.
But just for now as a hint you're probably gonna need an assert statement at one point so a test can pass or fail. We should also probably use a mock object, but more on that later :)